test/e2e-tests/tests/sanity_test.go

```diff
@@ -1,19 +1,68 @@
 package tests
 
 import (
-	"context"
+	"github.com/opendatahub-io/ai-edge/test/e2e-tests/oc"
 	support "github.com/opendatahub-io/ai-edge/test/e2e-tests/support"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"testing"
 )
 
-func TestGetAllPods(t *testing.T) {
+const (
+	ContainerImagePipeline    = "../../../pipelines/tekton/build-container-image-pipeline/"
+	ContainerImagePipelineRun = "../../../pipelines/tekton/build-container-image-pipeline/build-container-image-pipelinerun-bike-rentals.yaml"
```

Comment:
This works but of course it is not ideal. I am not really sure how else we want to specify where out pipelines and pipelineRuns are located. 

Maybe though an envriment variable which is set the Makefile? I am not sure